batch_correction/README.md

```diff
@@ -0,0 +1,8 @@
+# refine.bio Examples: Batch Correction
```

Comment:
I don't think this README is as intended. The text under both headers is pretty similar and the H3 header is from another example.